src/Workspaces/Core/Portable/Workspace/DocumentActiveContextChangedEventArgs.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis
+{
+    public sealed class DocumentActiveContextChangedEventArgs : EventArgs
+    {
+        public Workspace Workspace { get; }
+        public SourceTextContainer SourceTextContainer { get; }
+        public DocumentId PreviousContextDocumentId { get; }
+        public DocumentId NewContextDocumentId { get; }
```

Comment:
The only naming issue I see is the mixed combination of using previous and new..  Either use old/new or previous/current (or next, but that doesn't make sense here.)